x/protorev/keeper/rebalance.go

```diff
@@ -153,5 +153,10 @@ func (k Keeper) ExecuteTrade(ctx sdk.Context, route swaproutertypes.SwapAmountIn
 		return err
 	}
 
+	// Update the developer fees
+	if err = k.UpdateDeveloperFees(ctx, inputCoin, tokenOutAmount); err != nil {
+		return err
+	}
```

Comment:
hrmm, in statistics its based on `	profit := tokenOutAmount.Sub(inputCoin.Amount)`

shouldn't that be whats passed in here as well?

Reply:
Ah I see its calculated again in the function, missed that.

Reply:
I'll clean it up so the same profit obj is passed around instead of recalculating it.

Reply:
updated the PR so we use a single profit calc and opt for passing only denom instead of the input coin for both `UpdateStatistics` and `UpdateDeveloperFees`.